### ISSUE TITLE
UI: Create EM Submission Table Layout

### ISSUE THREAD

**Issue by acollad1** [US-EPA-CAMD/PI-1#5216]
As an ECMPS admin, I want to see the EM Submission Table on the EM Submission screen so that I can support program goals.

Given an EPA admin has logged into the ECMPS application
When the user navigates to the EM Submission screen via the left-hand navigation
Then the user will see the EM Submission action buttons Open, Extend, Close, and Approve 

Given an EPA admin has logged into the ECMPS application
When the user navigates to the EM Submission screen via the left-hand navigation
Then the user will see the EM Submission table shell UI displayed with data

https://xd.adobe.com/view/5f899592-74d7-4284-9fbf-c500c91134b8-0ada/

**Comment by fanwari1** [comment]
NOTE TO TESTER:
There are some columns that we are to skip, which came from our spike ticket [5264](https://github.com/US-EPA-CAMD/easey-ui/issues/5264):
We are skipping:
```
Last Window
Submission Status
Submission Date
Representative
Analyst
```

**Comment by mosesdeeCVP** [comment]
![Create EM Submission Table Layout 5216.png](https://images.zenhubusercontent.com/5f807025564a417cbcdf6d69/23db4b2f-f04e-43e6-99b6-fd118bc15cf9)

**Comment by mosesdeeCVP** [comment]
Apply Filter button does not work as of yet. This will be addressed in ticket [#5225](https://app.zenhub.com/workspaces/ecmps-prod-team-64623d2269329d0c96ba04f3/issues/gh/us-epa-camd/easey-ui/5225). Table will be tested there.

**Comment by mosesdeeCVP** [comment]
EM Submission Table Layout has be developed
